AntennaTracker/Tracker.cpp

```diff
@@ -23,37 +23,38 @@
 #include "version.h"
 #undef FORCE_VERSION_H_INCLUDE
 
-#define SCHED_TASK(func, _interval_ticks, _max_time_micros) SCHED_TASK_CLASS(Tracker, &tracker, func, _interval_ticks, _max_time_micros)
```

Comment:
I probably would have made a lower number a higher priority.. I guess it's arbitrary though.

Reply:
I too think lower number being higher priority is more conventional

Reply:
Changed it to be lower-is-higher-priority, thanks.